IntroToCSharp/Pages/Activities/Variables.razor

```diff
@@ -0,0 +1,148 @@
+    <MudList Clickable="true">
+        <MudListItem Icon="@Icons.Material.Filled.Badge" IconColor="Color.Secondary" Text="Name">
+            <NestedList>
+                <MudListItem>
+                    <MudText>This is the “address” that you declare that allows you to access the variable later.
+                    </MudText>
+                </MudListItem>
+            </NestedList>
+        </MudListItem>
+        <MudListItem Icon="@Icons.Material.Filled.Category" IconColor="Color.Warning" Text="Type">
+            <NestedList>
+                <MudListItem>
+                    <MudText>The type refers to the data being stored. Data can be a bunch of different things so it’s
+                        important for the computer to know what type of data it is handling.
+                    </MudText>
+                    <Example Title="Ice Cream Truck">
+                        You are not going to go up to the ice cream truck and just say you want ice cream
+                        because the vender won’t know what flavor to give you. Instead, you tell him what flavor or type
+                        of ice cream you want. That way he can go straight to the freezer and get strawberry ice cream.
+                    </Example>
+                </MudListItem>
+            </NestedList>
+        </MudListItem>
+        <MudListItem Icon="@Icons.Material.Filled.Storage" IconColor="Color.Tertiary" Text="Data">
+            <NestedList>
+                <MudListItem>
+                    <MudText>Data is the actual information stored.</MudText>
+                    <Example Title="Fun Fact">
+                        Let's say that you memorize the date when the United States declared its
+                        independence. The "data" stored in your brain would be "July 4, 1776".
+                    </Example>
+                </MudListItem>
+            </NestedList>
+        </MudListItem>
+    </MudList>
+
+</Section>
+
+<Section Title="Using Variables">
+
+    <Note>We will learn what Console.ReadLine() and Console.WriteLine() do later.</Note>
+    <CodeBlock Filename="Variables/Test.cs" ReplIt="JoshuaJoshJosh1/VariablesExample" />
+    <MudText>
+        In order to declare a variable, you must first list the type (Ex: string, int, double, char, etc.) and then the
+        chosen name of the variable. Once you have declared a variable, you can use the variable's name in an expression. 
+        If you do not declare your variables beforehand, the program will not function.
+    </MudText>
+    <Note>
+        For proper formatting, variables are usually are placed at the very beginning of the program
+    </Note>
+    <MudText>
+        When writing a program, you will have many variables. Each one can be initialized only once, but can be
+        declared/reused
+        numerous times.
+    </MudText>
+</Section>
+
+<Section Title="More on Naming Variables">
+    <SubSection Title="The Rules">
```

Comment:
I love this! I would add in a short paragraph above it saying something like, "When programming, variables are essential for managing complexity. They are used to create labels that the programmer can use to store, update, and read data. You have a lot of flexibility in what you name your variables but there are a few restrictions. Below are the rules for naming a variable in C#: